tests/unit/checkpoint/test_pipeline.py

```diff
@@ -0,0 +1,119 @@
+from deepspeed.runtime.checkpoint_engine.torch_checkpoint_engine import TorchCheckpointEngine
+from tests.unit.common import DistributedTest
+from tests.unit.simple_model import *
+
+from tests.unit.checkpoint.common import checkpoint_correctness_verification
+
+import pytest
+
+
+@pytest.fixture(params=[2])
+def stages(request):
+    return request.param
```

Comment:
What's the purpose of having this be a fixture? Is there a reason we may want to change the value of `stages` later?

Reply:
I thought it was the right way to reproduce a default argument value as done in the original [test](https://github.com/microsoft/DeepSpeed/blob/f82846d7df6a129668bea447a0f953f6cf345efb/tests/unit/test_checkpointing.py#L661). Will remove and simplify

Reply:
Done

Reply:
That makes sense, I think it's probably fine either way!